scipp_widgets/inputs.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright (c) 2020 Scipp contributors (https://github.com/scipp)
+# @file
+# @author Matthew Andrew
+import ipywidgets as widgets
+
+
+class Input():
+    def __init__(self,
+                 name,
+                 validator=lambda input: input,
+                 options=(),
+                 tooltip=''):
+        self.name = name
+        self.validator = validator
+        self._options = options
+        self._tooltip = tooltip if tooltip else name
+
+    def create_input_widget(self):
+        return widgets.Combobox(placeholder=self._tooltip,
+                                continuous_update=False,
+                                options=self._options)
+
+
+class EvalInput():
```

Comment:
I'm thinking that `EvalInput` should just be an `Input`, they are so similar.
Also easier for users to only remember one type?

Reply:
I think they need to be different types. If we had just EvalInput you would not be able to enter a string without adding the quotes around it. For example with the summation example you would have to enter `'tof'` rather than `tof`. Unless we want to require that people always have to enter python expressions valid in the notebooks scope?

Reply:
Upon consideration I think you are right here. I've added the eval_input as a kwarg option instead.